Content.Shared/Wieldable/WieldableSystem.cs

```diff
@@ -90,6 +90,8 @@ private void OnGunRefreshModifiers(Entity<GunWieldBonusComponent> bonus, ref Gun
         {
             args.MinAngle += bonus.Comp.MinAngle;
             args.MaxAngle += bonus.Comp.MaxAngle;
+            args.AngleDecay += bonus.Comp.AngleDecay;
+            args.AngleIncrease += bonus.Comp.AngleIncrease;
```

Comment:
This is good to keep but this should go on an upstream PR since it modifies an upstream file.
This is probably a better solution than just cranking up the camera recoil very high alone, for one because it doesn't make people throw up and two because a modified client cannot cheat modified angle values on the gun.
From the videos on Discord though the unwielded spread is too high, this is how shooting an M41A in cm13 looks unwielded vs wielded with a firearms skill of 1 (rifleman):

https://github.com/CM-14/CM-14/assets/10968691/5511b16d-185b-4c33-be0a-de7b3344425f

With a firearms skill of 0 it is a lot worse depending on where you are aiming and hard to see, but the accuracy penalty is less severe than in the videos since it doesn't quite shoot sideways:

https://github.com/CM-14/CM-14/assets/10968691/4f64c34c-2d0f-496c-8bf2-1ed9f25a0e8f